.env-defaults

```diff
@@ -0,0 +1,32 @@
+# Container entrypoints will run database migrations if set to "true"
+DATABASE_MIGRATION=true
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG=True
+
+# SECURITY WARNING: enable two factor authentication in production!
+TWOFACTOR_ENABLED=False
+
+# --- OpenTracing --- #
```

Comment:
OpenTelemetry, not OpenTracing (its precursor)

Reply:
Fixed!